pkg/skaffold/runner/v1/new.go

```diff
@@ -133,9 +136,7 @@ func NewForConfig(runCtx *runcontext.RunContext) (*SkaffoldRunner, error) {
 		return nil, fmt.Errorf("creating watch trigger: %w", err)
 	}
 
-	podSelectors := kubernetes.NewImageList()
-
-	rbuilder := runner.NewBuilder(builder, tagger, artifactCache, podSelectors, runCtx)
+	rbuilder := runner.NewBuilder(builder, tagger, artifactCache, podSelector, runCtx)
```

Comment:
do we still need a podSelector in a Builder?

Reply:
whoops meant to remove. thanks